app/assets/stylesheets/preact/article-form.scss

```diff
@@ -41,20 +42,21 @@
   width: 97%;
   max-width: 860px;
   padding: 0px 6px;
-  background: white;
-  background: var(--theme-container-background, white);
+  @include dev-card;
+  // background: white;
```

Comment:
If the code is no longer used, delete it. We have git.

Reply:
100% agree. I'll clean it up a little.